src/ngCookies/cookieStore.js

```diff
@@ -5,6 +5,8 @@ angular.module('ngCookies').
  * @ngdoc service
  * @name $cookieStore
  * @deprecated
+ * Please use the {@link ngCookies.$cookies `$cookies`} service instead.
```

Comment:
Hasn't this been deprecated for some time now? Perhaps we should schedule a removal timeline?